clients/graphql/genqlient.graphql

```diff
@@ -53,11 +53,13 @@ query getAlerts($input: AlertsInput) {
                     hash
                     botId
                     timestamp
+                    chainId
```

Comment:
does this require changes on the graphql api?

Reply:
that's already merged. Related PR:
https://github.com/forta-network/forta-alerts-api/pull/36